backend/routers/energy.py

```
@@ -0,0 +1,105 @@
+from database import get_db
+from models import Meter, Consumption
+import schemas
+from auth import get_current_user
+
+router = APIRouter(prefix="/api/energy", tags=["energy"])
+
+# Meters
+@router.post("/meters")
+async def create_meter(
+    meter: schemas.MeterCreate,
+    db: Session = Depends(get_db),
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)